src/common/schema/application.js

```diff
@@ -47,19 +57,12 @@ module.exports = {
           type: 'string',
           minLength: 1,
           maxLength: 51
-        },
```

Comment:
Oh because they're the only ones that are consistent across the 2 options for country fields?